xcube_zenodo/store.py

```diff
@@ -20,44 +20,63 @@
 # SOFTWARE.
 
 import logging
+import os
 from typing import Tuple, Iterator, Container, Any, Union
 
 import requests
 import xarray as xr
 from xcube.util.jsonschema import (
+    JsonBooleanSchema,
     JsonObjectSchema,
     JsonStringSchema,
 )
 from xcube.core.store import (
     DataDescriptor,
     DataStore,
+    DataStoreError,
     DataTypeLike,
     new_data_store,
 )
 
 from .constants import API_RECORDS_ENDPOINT
+from .constants import COMPRESSED_FORMATS
+from .constants import PRELOAD_CACHE_FOLDER
+from .preload import PreloadHandle
+from ._utils import identify_file_format
 from ._utils import get_attrs_from_record
 from ._utils import is_supported_file_format
-from ._utils import translate_data_id2uri
+from ._utils import is_supported_compressed_file_format
+from ._utils import translate_data_id2fs_path
 
 
-_LOG = logging.getLogger("xcube")
+LOG = logging.getLogger(__name__)
```

Comment:
This will evaluate to `logging.getLogger("store")`. To unspecific, use logger name "xcube-zenodo" instead.